README.md

```diff
@@ -69,6 +69,35 @@ let cryptoDecorator = try VaultProviderFactory.createLegacyVaultProvider(from: m
 
 :warning: This library supports vault version 6 and higher.
 
+### Box
+
+Modify your app delegate as described in [Box iOS SDK](https://github.com/box/box-ios-sdk). In addition, the following constants must be set once, e.g. in your app delegate:
+
+```swift
+let clientId = ... // your Box client identifier
+let clientSecret = ... // your Box client secret
+let sharedContainerIdentifier = ... // optional: only needed if you want to create a `BoxCloudProvider` with a background `URLSession` in an app extension 
+BoxSetup.constants = BoxSetup(clientId: clientId, clientSecret: clientSecret, sharedContainerIdentifier: sharedContainerIdentifier)
+```
+
+Begin the authentication flow:
+
+```swift
+let tokenStore = BoxTokenStore()
+let viewController = ... // the presenting `UIViewController`
+BoxAuthenticator.authenticate(credential: credential, from: viewController).then { 
```

Comment:
Creating the `BoxCredential` is missing here. :wink:

Reply:
Fixed in commit 995defc.